pom.xml

```diff
@@ -23,7 +23,7 @@
       <java.version>11</java.version>
       <maven.compiler.source>${java.version}</maven.compiler.source>
       <maven.compiler.target>${java.version}</maven.compiler.target>
-      <activiti-cloud-runtime-bundle-service.version>7.1.209</activiti-cloud-runtime-bundle-service.version>
+      <activiti-cloud-runtime-bundle-service.version>7.1.0-PR-453-53-SNAPSHOT</activiti-cloud-runtime-bundle-service.version>
```

Comment:
@igdianov, is this intended to be merged or only a temporary change to create a preview environment?

Reply:
@erdemedeiros Yes, this temporary to create preview environments using CI in PRs. It will be updated using release versions after it is merged.